grammar and readability changes

iiitttsss opened the pull request.

iiitttsss committed:
Update L01_fully_connected_neural_networks.md

grammar and readability changes

iiitttsss committed:
Update L01_fully_connected_neural_networks.md

grammar and readability changes

Cphyr commented on Datascience101/chapter_03_machine_learning_2/lessons/L01_fully_connected_neural_networks.md:
```diff
@@ -8,33 +8,35 @@ Tags: Done
 ## Linear Layer
 
-Let us think of performing multiple logistic regressions in parallel, each with it's own weights vector and bias, and finally applying the activation function on separately each regression.
+Lets think about performing multiple logistic regressions in parallel, each with its own weight vector and bias, and then applying the activation function on each of these.
```
Lets is not an English word. You may say Let us or its short version Let's

Cphyr reviewed (changes requested):
@iiitttsss

also, don't forget to follow the PR guidelines. You must open an issue regarding this PR and link it to this PR

iiitttsss committed:
Merge branch 'Cphyr:main' into main

TicTalTech committed:
fixed let's
added a "Motivation" paragraph
added more information about activation functions
